action/action.go

```diff
@@ -78,7 +79,7 @@ func (a Action) Run(c claim.Claim, creds credentials.Set, opCfgs ...OperationCon
 	}
 
 	var opErr *multierror.Error
-	opResult, err := a.Driver.Run(op)
+	opResult, err := a.Driver.Run(context.TODO(), op)
```

Comment:
I'm still working on how we want to introduce the context, leaning towards the first parameter to `Action.Run` unless there's a reason for opCfgs to be involved somehow?

Reply:
Making context the first parameter is probably most idiomatic. From the [package docs](https://golang.org/pkg/context/):

> Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx